### PR TITLE
[ICP-13108] Delay off event, add chime when tamper is triggered

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @dkirker @SmartThingsCommunity/srpol-pe-team 
Could you check it?

### CODE REVIEW

File: devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy [hunk 1]
```diff
@@ -206,7 +206,8 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 			case 0x09: //TAMPER
 				sendEvent(name: "tamper", value: "detected")
 				sendEvent(name: "alarm", value: "both")
-				runIn(2, "clearTamperAndAlarm")
+				sendEvent(name: "chime", value: "chime")
```
Comment (dkirker) [inline devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:209]: Do we actually want to send `chime` here? It should be safe to still clear it with `clearTamperAlarmAndChime`.

Comment (greens) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:209]: I assume we want to send chime wherever we're sending alarm?

Comment (MGoralczykS) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:209]: @greens Yes, in my opinion, this is correct.

Comment (dkirker) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:209]: But a chime isn't an alarm. That said, the device isn't actually integrated in SmartThings as it was designed, and if I recall one of the endpoints is associated with tamper (I think the first one, actually). So I guess for this integration chime == alarm == tamper, which makes no sense at all.

**Comment by dkirker** [comment]
So does the device send us "off" before it is even finished? Is it likely that this is happening because the device integration is actually misusing the endpoints and the endpoint that is sending is the really quick off is something more short term like the doorbell noise instead of an alarm?

**Comment by greens** [comment]
So, @dkirker this was related to an earlier ticket that was raised where when the device is tampered, the alarms will go off, but to the device this is a different alarm and so it does not signal that the alarms are on. Cert did not like that you could have a situation where the alarms were going off without the state being changed.

So we added this event which automatically clears itself to encapsulate the behavior when the alarms are triggered by tamper. Physically, when tampered, the device will sound its alarms for 2s and then turn them off. No indication is otherwise given to us beyond the tamper notification.

**Comment by k-morawski** [comment]
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/49092

**Pull request closed without merging**